src/screens/authenticationScreens/ForgotPasswordScreen.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import {
+	Alert,
+	SafeAreaView,
+	ScrollView,
+	StyleSheet,
+	Text,
+	View
+} from "react-native";
+import PropTypes from "prop-types";
+import LottieView from "lottie-react-native";
+
+import {
+	COLOR_PALETTE,
+	DEFAULT_FORM_VALUES,
+	FORGOT_PASSWORD_INPUTS_ARRAY
+} from "../../utils/constants";
+import Form from "../../components/common/form/Form";
+import GlobalStyles from "../../utils/GlobalStyles";
+import { handleForgotPassword } from "../../utils/firebase";
+
+const ForgotPasswordScreen = ({ navigation }) => {
+	const [isLoading, setIsLoading] = useState(false);
+
+	const onSubmit = async (data) => {
+		setIsLoading(true);
+		await handleForgotPassword(data, () => {
+			Alert.alert(
+				"Password Reset",
+				"Link to reset password has been sent to your email"
+			);
+			navigation.goBack();
+		});
+	};
+	return (
+		<SafeAreaView style={[StyleSheet.absoluteFill, GlobalStyles.appView]}>
+			<ScrollView style={styles.wrapper}>
+				<View style={styles.container}>
+					<LottieView
+						source={require("../../assets/lottie/signInAnimation.json")}
```

Comment:
When you copy and paste, make sure you change animations and make the effort to make each page look a bit different. That way we have more to show. A different animation here, to learn how to work with lottie I think is good practice for you